src/Bundle/DependencyInjection/SyliusGridExtension.php

```diff
@@ -60,6 +62,17 @@ public function load(array $configs, ContainerBuilder $container): void
             ->addTag('sylius.grid')
         ;
 
+        $container->registerAttributeForAutoconfiguration(AsFilter::class, function (ChildDefinition $definition, AsFilter $attribute, \Reflector $reflector): void {
+            if (!$reflector instanceof \ReflectionClass) {
```

Comment:
Do you still need to use reflection if you've specified target on the attribute?

![image](https://github.com/user-attachments/assets/3012b292-591d-4fab-89f3-e5015bea3b04)

Reply:
I did this for static analysis. \Reflector->getName() does not exist, that's why I ensure we uses the \ReflectionClass. Any better idea ?

Reply:
yes, @vasilvestre specified the attribute to be used only on class.